tests/test_load_encyclopedia_extra.py

```diff
@@ -0,0 +1,313 @@
+
+	</div>
+</nav>
+
+
+</div>
+
+</div>
+
+<footer id="footer" class="mw-footer" role="contentinfo" >
+	<ul id="footer-info">
+	<li id="footer-info-lastmod"> This page was last edited on 27 May 2011, at 09:07.</li>
+</ul>
+
+	<ul id="footer-places">
+	<li id="footer-places-privacy"><a href="/index.php?title=The_Encyclopedia_of_Mormonism:Privacy_policy">Privacy policy</a></li>
+	<li id="footer-places-about"><a href="/index.php?title=The_Encyclopedia_of_Mormonism:About">About The Encyclopedia of Mormonism</a></li>
+	<li id="footer-places-disclaimer"><a href="/index.php?title=The_Encyclopedia_of_Mormonism:General_disclaimer">Disclaimers</a></li>
+</ul>
+
+	<ul id="footer-icons" class="noprint">
+	<li id="footer-poweredbyico"><a href="https://www.mediawiki.org/"><img src="/resources/assets/poweredby_mediawiki_88x31.png" alt="Powered by MediaWiki" srcset="/resources/assets/poweredby_mediawiki_132x47.png 1.5x, /resources/assets/poweredby_mediawiki_176x62.png 2x" width="88" height="31" loading="lazy"/></a></li>
+</ul>
+
+</footer>
+
+<script>(RLQ=window.RLQ||[]).push(function(){mw.config.set({"wgPageParseReport":{"limitreport":{"cputime":"0.009","walltime":"0.011","ppvisitednodes":{"value":10,"limit":1000000},"postexpandincludesize":{"value":0,"limit":2097152},"templateargumentsize":{"value":0,"limit":2097152},"expansiondepth":{"value":2,"limit":100},"expensivefunctioncount":{"value":0,"limit":100},"unstrip-depth":{"value":0,"limit":20},"unstrip-size":{"value":0,"limit":5000000},"timingprofile":["100.00%    0.000      1 -total"]},"cachereport":{"timestamp":"20231109075949","ttl":86400,"transientcontent":false}}});mw.config.set({"wgBackendResponseTime":143});});</script>
+</body>
+<!-- Cached 20231109134817 -->
+</html>
+"""
+
+
+def test_load_encyclopedia() -> None:
+    """It returns a valid Document for a conference talk."""
+    url = "https://eom.byu.edu/index.php?title=%22Fundamentalists%22"
+    result = load_encyclopedia.load_encyclopedia(url, html)
+    assert len(result.page_content) > 0
+    assert result.metadata["url"] == url
+    assert result.metadata["title"] == '"Fundamentalists"'
+    assert result.page_content.startswith("[See this page")
+    assert not result.page_content.startswith("[A](/index.php?")
```

Comment:
It looks like you're trying to remove everything after [A] index.php right? If so then can you change this test to the following to make sure that [A] (/index.php?) isn't anywhere in the page content?

"[A](/index.php?" not in result.page_content